tests/test_validate_outputs.py

```diff
@@ -504,7 +505,8 @@ def test_skycorr(self):
         self.check_datasets(["skyCorr"], self._num_exposures)
 
     def check_aperture_corrections(self, catalog, **kwargs):
-        for alg in ("base_PsfFlux", "base_GaussianFlux"):
+        names = kwargs.get("aperture_algorithms", ("base_PsfFlux", "base_GaussianFlux"))
```

Comment:
I think that the intention here was that the kwargs that are supported should be in the function signature, as in `check_psf_stars_and_flags`.  We still need the overflow `**kwargs` because of the way it's being called in general, though.